extensions/coap/tests/CMakeLists.txt

```diff
@@ -40,7 +40,7 @@ FOREACH(testfile ${CURL_INTEGRATION_TESTS})
 	target_include_directories(${testfilename} BEFORE PRIVATE "../../http-curl/sitetosite/")
 	target_include_directories(${testfilename} BEFORE PRIVATE "${CMAKE_SOURCE_DIR}/extensions/civetweb/")
 	target_include_directories(${testfilename} BEFORE PRIVATE ./include)
-    createTests("${testfilename}")
+        createTests("${testfilename}")
```

Comment:
The indentation is still inconsistent: the rest of the file uses tabs while this line now uses 8 spaces instead of 4.

Reply:
Yep, that's fair, edited outside my normal IDE, fixed.